docs/source/developer/contributing.rst

```diff
@@ -288,7 +288,7 @@ about 7 minutes again.
 Setting up a local development environment
 ------------------------------------------
 This section explains how to set up a local development environment. We assume you use GNU/Linux,
-macOS, or Windows Subsystem for Linux.
+macOS, or Windows Subsystem for Linux. If using Windows, we recommend installing `Anaconda for windows <https://www.anaconda.com/download>`__ and then using the Anaconda command prompt for all installation steps.
```

Comment:
Could you rebase your branch to separate the changes as this is part of another PR (that got merge - congrats for that).